pkg/api/unversioned/types.go

```diff
@@ -298,6 +298,8 @@ const (
 // +protobuf.options.(gogoproto.goproto_stringer)=false
 type APIVersions struct {
 	TypeMeta `json:",inline"`
+	// public hostname of the server that is serving this group.
+	PublicHostname string `json:"publicHostname"`
```

Comment:
I'd like to be consistent with service terminology, which is "External" rather than "Public":

https://github.com/kubernetes/kubernetes/blob/master/pkg/api/v1/types.go#L1740